src/main/java/apoc/export/json/JsonFormat.java

```diff
@@ -0,0 +1,139 @@
+package apoc.export.json;
+
+import apoc.export.json.serialize.CustomSerializeNode;
+import apoc.export.json.serialize.CustomSerializeRelationship;
+import apoc.export.util.ExportConfig;
+import apoc.export.util.Format;
+import apoc.export.util.Reporter;
+import apoc.result.ProgressInfo;
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.neo4j.cypher.export.SubGraph;
+import org.neo4j.graphdb.*;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.Writer;
+import java.util.Iterator;
+import java.util.List;
+
+import static apoc.util.JsonUtil.getPropertiesArray;
+
+public class JsonFormat implements Format {
+    private final GraphDatabaseService db;
+
+    public JsonFormat(GraphDatabaseService db) {
+        this.db = db;
+    }
+
+    @Override
+    public ProgressInfo load(Reader reader, Reporter reporter, ExportConfig config) throws Exception {
+        return null;
+    }
+
+    @Override
+    public ProgressInfo dump(SubGraph graph, Writer writer, Reporter reporter, ExportConfig config) throws Exception {
+        try (Transaction tx = db.beginTx()) {
+            writeAll(graph, reporter, config, writer);
+            tx.success();
+            writer.close();
+            return reporter.getTotal();
+        }
+    }
+
+    public ProgressInfo dump(Result result, Writer writer, Reporter reporter, ExportConfig config) throws Exception {
+        try (Transaction tx = db.beginTx()) {
+            ObjectMapper mapperNode = new ObjectMapper();
+            ObjectMapper mapperRel = new ObjectMapper();

```

Comment:
create a private class `JsonCustomMappers` (inside this class) and implement a static method called `createMapperForRels`, so that:

ObjectMapper mapperNode = JsonCustomMappers.createMapperForRels();